tests/query_workflow_test.go

```diff
@@ -291,9 +293,11 @@ func (s *clientIntegrationSuite) TestQueryWorkflow_QueryFailedWorkflowTask() {
 	s.NotNil(workflowRun)
 	s.True(workflowRun.GetRunID() != "")
 
-	// wait for workflow to fail
-	time.Sleep(time.Second * 2)
+	// wait for workflow task to fail 3 times
+	time.Sleep(time.Second * 15) // 1st_attempt, 0_delay, 2nd_attempt, 10s_delay, 3rd_attempt
```

Comment:
😮 that seems long, is it possible to specify a lower workflow task timeout on start so we don't need to wait that long?